docker-compose.yml

```diff
@@ -0,0 +1,106 @@
+name: harmony-dev
+services:
+  postgres:
+    image: postgres:13.7-alpine
+    restart: always
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: zenysis
+      POSTGRES_DB: postgres
+      POSTGRES_HOST_AUTH_METHOD: trust
+    ports:
+      - "5432:5432"
+    volumes:
+      - pgdata-volume:/var/lib/postgresql/data
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready -U postgres -d postgres" ]
+      start_period: 0s
+      interval: 10s
+      timeout: 30s
+      retries: 3
+  redis:
+    image: redis
+    ports:
+      - 6379:6379
+    healthcheck:
+      test: [ "CMD", "redis-cli", "--raw", "incr", "ping" ]
+      interval: 10s
+      timeout: 30s
+      start_period: 0s
+      retries: 3
+  hasura:
+    image: hasura/graphql-engine:v2.11.3.cli-migrations-v2
+    ports:
+      - 8088:8080
+    environment:
+      HASURA_GRAPHQL_ENABLE_TELEMETRY: "false"
+      HASURA_GRAPHQL_ENABLE_CONSOLE: "true"
+      HASURA_GRAPHQL_MIGRATIONS_SERVER_TIMEOUT: 500
+      HASURA_GRAPHQL_DATABASE_URL: postgresql://postgres:zenysis@postgres:5432/${ZEN_ENV}-local
+      HASURA_GRAPHQL_METADATA_DIR: '/hasura-metadata/latest'
```

Comment:
as discussed on slack, removing `HASURA_GRAPHQL_METADATA_DIR` and `HASURA_GRAPHQL_MIGRATIONS_DIR` here as well as `./scripts/db/hasura/dev/container:/zenysis:ro`. and modifying `./graphql/hasura/metadata/versions:/hasura-metadata` -> `./graphql/hasura/metadata/versions/latest:/hasura-metadata`

Reply:
HASURA_GRAPHQL_METADATA_DIR actually needs to stay
HASURA_GRAPHQL_MIGRATIONS_DIR can be removed - if not present migrations are skipped

./graphql/hasura/metadata/versions:/hasura-metadata is the correct mapping - so not changing that.

Reply:
It's not particularly important, but to explain my comment more.

The default `HASURA_GRAPHQL_METADATA_DIR` is `/hasura-metadata`.  So I was saying if we change the volume to point `/hasura-metadata` at the latest folder in versions like `./graphql/hasura/metadata/versions/latest:/hasura-metadata`. Then you don't need to additional set the `HASURA_GRAPHQL_METADATA_DIR` to be that latest folder because it's already pointing to it.

Reply:
Aaah - ok - got it - I'll make that change on https://github.com/Zenysis/Harmony/pull/83